TeslaHW/TeslaHW/Charging/ChangeProgressShape.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  ChangeProgressShape.swift
+//  TeslaHW
+//
+//  Created by Александр Троицкий on 04.02.2023.
+//
+
```

Comment:
```suggestion
//
//  ChangeProgressShape.swift
//  TeslaHW
//
//  Created by Александр Троицкий on 04.02.2023.
//

```
кто-то отключил форматтер)